features/notifications-apps.yml

```
@@ -0,0 +1,19 @@
+name: Notifications from service workers and installed apps
+# This feature is for the platform-specific nature of notifications support
+# in Chrome Android and Safari iOS, and anchors to the Caniuse feature.
+#
+# See `features/notifications.yml` for the full Notifications feature, which
+# does not align with Caniuse.
+description: "The Notifications API displays messages to the user, with platform-specific requirements in some cases. Chrome Android supports notifications from installed service workers, and Safari iOS supports notifications from websites that have been bookmarked on the homescreen."
```

```suggestion
description: The Notifications API, when used from a service worker or home screen app, displays messages to the user, typically using the notification system of the host operating system. Chrome Android supports notifications from installed service workers, and Safari iOS supports notifications from websites that have been added to the home screen."
```

See my other comment. I used the exact same description that was in the general feature, but added the "when used from ..." caveat. And I kept the follow-up about the chrome and safari specifics.

Minor: I changed "bookmarked" with "added" for Safari, because Safari iOS uses the verb "add" in its UI: "Add to Home Screen".

---

Hm, need to shave 38 chars, i'll remix.

---

Oh, or not... passed tests locally.

---

Nope, I just hadn't pulled. Pushed a shorter version. Not great, but has the same bits there.